vox.gemspec

```diff
@@ -34,12 +34,12 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'mime-types', '~> 3.3.1'
   spec.add_runtime_dependency 'multi_json', '~> 1.15.0'
   spec.add_runtime_dependency 'websocket-driver', '~> 0.7.3'
-  spec.add_development_dependency 'rake', '~> 12.0'
+  spec.add_development_dependency 'rake', '~> 13.0'
   spec.add_development_dependency 'rspec', '~> 3.0'
-  spec.add_development_dependency 'rubocop', '~> 0.89.1'
-  spec.add_development_dependency 'rubocop-performance', '~> 1.7.1'
-  spec.add_development_dependency 'rubocop-rspec', '~> 1.42.0'
-  spec.add_development_dependency 'simplecov', '~> 0.17.1'
```

Comment:
Unfortunately, there's some weirdness with simplecov >17 which makes it break for most reporters. We're stuck here until code climate updates support for it.

Reply:
Yup, see that now. Reverted that change.